Scripts/application.py

```diff
@@ -0,0 +1,112 @@
+import sqlite3
+import json
+from typing import Dict
+from flask import Flask, request
+app = Flask(__name__)
+
+path_to_db = '../db/allYourSantaAreBelongToUs.db'
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+    
+@app.route('/api/v1/person', methods = ["GET","POST"])
+def return_all_people():
+    
+    if request.method == "POST":
+        posted_email = request.get_json()['email']

```

Comment:
email might be missing. Please verify there is a json to begin with and that it has an email key and if not return a 400 error code to the caller with a helpful message.

Reply:
This is done.